XbSymbolDatabase.c

```diff
@@ -120,6 +121,9 @@ SymbolDatabaseList SymbolDBList[] = {
     // DSOUNDH is just meant to define hot fix, there is no separate section
     //{ XbSymbolLib_DSOUNDH,{ Sec_DSOUND }, &DSound_OOVPAV2, DSound_OOVPA_COUNT },
 
+    // Only used in Chihiro titles and dashboard
```

Comment:
Make clear this applies to Chihiro dashboard, not Xbox dashboard: "and Chihiro dashboard".

(dashboard is also a wrong name here, so ideally replace that with the appropriate name [segaboot?])

Reply:
Chihiro applications is better to use. Plus updated the commit to use the reference.